config/application.rb

```diff
@@ -38,3 +38,20 @@ class Application < Rails::Application
     config.api_only = true
   end
 end
+
+# Rotate SHA1 cookies to SHA256 (the new Rails 7 default)
+# TODO: Remove this after existing user sessions have been rotated
+# https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#key-generator-digest-class-changing-to-use-sha256
+Rails.application.config.action_dispatch.cookies_rotations.tap do |cookies|
```

Comment:
I think we can probably drop this commit, as I don't think we set any cookies in this app (and we haven't got any users on here yet)